src/Frontend/test/specs/monitoring/sorting-endpoints.spec.ts

```diff
@@ -477,22 +482,22 @@ describe("FEATURE: Endpoint sorting", () => {
       await sortEndpointsBy({ column: columnName.CRITICALTIME }); // Act: Click the column title once for ascending
 
       //Assert
-      await waitFor(() => expect(ungroupedEndpointNames().Endpoints).toEqual(["Endpoint3", "Endpoint1", "Endpoint2"]));
-      const ascendingAvgValues = await waitFor(() => smallGraphAverageValuesByColumn({ column: columnName.CRITICALTIME }));
-      await waitFor(() => expect(ascendingAvgValues).toEqual(["225", "350", "800"]));
+      expect(ungroupedEndpointNames().Endpoints).toEqual(["Endpoint3", "Endpoint1", "Endpoint2"]);
+      const ascendingAvgValues = await smallGraphAverageValuesByColumn({ column: columnName.CRITICALTIME });
+      expect(ascendingAvgValues).toEqual(["225", "350", "800"]);
     });
   });
 });
 
 async function assertSortImageState(column: string, direction: "up" | "down" | null) {
```

Comment:
```suggestion
function assertSortImageState(column: string, direction: "up" | "down" | null) {
```
Since await was removed from the calls async should be removed from the function, right?